Filed/Hypoxia.md

```diff
@@ -1,15 +1,26 @@
+# Dangers
+Hypoxia is extremely dangerous for pilots.  Side effects include impaired judgement and decision-making; decreased response time; visual impairment; and eventually, unconsciousness.  For these reasons, pilots need to be extremely vigilant to detect and avoid hypoxia.
+
+# Types of Hypoxia
+1. Hypoxic Hypoxia: Not enough available oxygen.  This is most common in pilots flying at higher altitudes
+2. Hypemic Hypoxia: Oxygen is available, but the body is not able to take it up.  Carbon Monoxide poisoning falls in this category.
```

Comment:
Might be worth noting there are four types of hypoxia: hypoxic hypoxia, hypemic hypoxia, stagnant hypoxia, and histotoxic hypoxia

More info here: https://www.faa.gov/pilots/training/airman_education/topics_of_interest/hypoxia

Reply:
Done.  See https://github.com/johnjackoleary/cfi-commonplace-book/pull/158